src/components/dropdown-select.js

```diff
@@ -180,21 +177,23 @@ const DropdownSelect = ({
   }
 
   const handleOnClick = (i, value) => {
+    let newSelectedOptions
```

Comment:
lets initialize the variable with something meaningful so we know type of value it will be receiving

Reply:
:heavy_check_mark: